libnmstate/nm/ipv6.py

```diff
@@ -15,10 +15,46 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
-from . import nmclient
+import socket
```

Comment:
Just a convention I prefer: Please separate std, 3rd party and internal packages using a newline.